core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -275,7 +266,7 @@ class AnalyzerManager(
         }
         def receive: Receive = {
           case FullTypeCheckCompleteEvent =>
-            analyzer ! RemoveFilesReq(filesToBeDeleted.toList.map(f => richEnsimeFile(f.file).toPath.toFile))
+            analyzer ! RemoveFilesReq(filesToBeDeleted.map(toFile))
```

Comment:
Because `TypeCheckFilesReq` would enter symbols into the `symsByFile` map and then I can delete those with `RemoveFilesReq`. 
I am actually wanting to delete the symbols that were deleted before we get a request for the project-wide actor.